assets/css/main.css

```diff
@@ -2,12 +2,25 @@
     line-height: 1.8;
     --theme-color: #7587ff;
     --base-font-family:  "Open Sans";
+    --base-font-size: 16px;
+    --base-font-family: "Segoe UI", Roboto, "Helvetica Neue", Arial;
```

Comment:
hmm not sure about the font choice though, Open Sans is the font we use for other branding stuffs